include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_utils.h

```diff
@@ -480,7 +480,7 @@ class __future : private std::tuple<_Args...>
     __wait_and_get_value(sycl::buffer<_T>& __buf)
     {
         //according to a contract, returned value is one-element sycl::buffer
-        return __buf.template get_access<access_mode::read>()[0];
+        return __buf.get_host_access()[0];
```

Comment:
```suggestion
        return __buf.get_host_access(sycl::read_only)[0];
```

Reply:
@timmiesmith @akukanov Thanks so much for the tip. I was trying with `sycl::access_mode::` and started to dig into the buffer/accessor constructors since it wasn't working.